spec/test-utils.js

```diff
@@ -12,10 +12,12 @@ const mockGithub = {
               title: 'My second fake issue'
             }]
           });
-        } else {
+        } else if (repo == 'nonexisting') {
```

Comment:
Does this mock method behave in the same way as how github handles its responses? What cases fall under the last case? Is it Like some network errors?